package.json

```diff
@@ -32,7 +32,7 @@
     "ejs": "^2.4.2",
     "express": "^4.14.0",
     "express-session": "^1.13.0",
-    "github": "^2.4.0",
```

Comment:
version 7 has significant API changes comparing to v2. Do not merge! Update API calls first